2024-09-28 22:25:53.092 WIB [13523] LOG:  starting PostgreSQL 14.13 (Homebrew) on aarch64-apple-darwin23.4.0, compiled by Apple clang version 15.0.0 (clang-1500.3.9.4), 64-bit
2024-09-28 22:25:53.093 WIB [13523] LOG:  listening on IPv6 address "::1", port 5432
2024-09-28 22:25:53.093 WIB [13523] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-09-28 22:25:53.094 WIB [13523] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2024-09-28 22:25:53.096 WIB [13524] LOG:  database system was shut down at 2024-09-28 22:25:39 WIB
2024-09-28 22:25:53.097 WIB [13523] LOG:  database system is ready to accept connections
2024-09-28 22:27:50.210 WIB [13766] FATAL:  database "api_db" does not exist
2024-09-28 22:29:15.765 WIB [13780] ERROR:  database "api_db" already exists
2024-09-28 22:29:15.765 WIB [13780] STATEMENT:  CREATE DATABASE api_db;
2024-09-28 22:30:22.942 WIB [13932] FATAL:  role "django_users" does not exist
2024-09-28 22:30:40.340 WIB [13984] FATAL:  role "django_users" does not exist
2024-09-28 22:31:21.925 WIB [13847] ERROR:  database "api_db" already exists
2024-09-28 22:31:21.925 WIB [13847] STATEMENT:  CREATE DATABASE api_db WITH OWNER django_users;
2024-09-28 22:32:32.322 WIB [14110] FATAL:  role "django_user" does not exist
